tests/controller-test.js

```diff
@@ -554,6 +555,43 @@ OO = {
     Html5Skin.findMainVideoElement.call(controllerMock, div);
     Html5Skin.findMainVideoElement.call(controllerMock, {0:videoElement});
 
+    describe('Controller testing skin initialization', function() {
+      // TODO
+      // This gets cleared up below in the "destroy" tests due to the way this
+      // test suite is currently set up. We need a lot of refactoring to fix this,
+      // so we're working around it for now
+      var mb = controllerMock.mb;
+      var messageBusSpy;
+
+      beforeEach(function() {
+        controllerMock.mb = mb;
+        messageBusSpy = sinon.spy(controllerMock.mb, 'publish');
+      });
+
+      afterEach(function() {
+        messageBusSpy.restore();
+      });
+
+      it('should show Start Screen after player created', function() {
+        Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', 'elementId', {});
+        expect(controllerMock.state.screenToShow).toBe(CONSTANTS.SCREEN.START_SCREEN);
+      });
+
+      it('should raise INITIAL_PLAY_REQUESTED event when toggling play/pause on Start Screen', function() {
+        Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', 'elementId', {});
+        Html5Skin.togglePlayPause.call(controllerMock);
+        expect(messageBusSpy.calledOnce).toBe(true);
+        expect(messageBusSpy.calledWith(CONSTANTS.CUSTOM_EVENTS.INITIAL_PLAY_REQUESTED)).toBe(true);
+      });
+
+      it('should publish INITIAL_PLAY after INITIAL_PLAY_REQUESTED', function() {
```

Comment:
I though about adding this, but there's no easy way of doing it with the current unit test setup. This would be like testing the message bus though, so this should technically be covered in mjolnir already.